docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst

```diff
@@ -0,0 +1,8 @@
+Vivarium Modeling Strategy - Lifestyle Intervention
+---------------------------------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+Decision tree and algorithm for lifestyle intervention
```

Comment:
see my comments on the vivarium_polypill.rst, I expect this doc to hit similar points relevant to lifestyle intervention.